BlockchainSdk/Blockchains/TON/TONWalletManager.swift

```diff
@@ -22,6 +22,7 @@ final class TONWalletManager: BaseManager, WalletManager {
     private let networkService: TONNetworkService
     private let txBuilder: TONTransactionBuilder
     private var isAvailable: Bool = true
+    private var jettonWalletAddressCache = [Token: String]()
```

Comment:
nit, как будто по гайдлайнам так не пишем )
```suggestion
    private var jettonWalletAddressCache: [Token: String] = [:]
```

Reply:
поправил